--- /dev/null
+++ b/Mathlib/Unsorry/N4PlusOneFactorOverSqrtShift.lean
@@ -0,0 +1,9 @@
+/-
+Copyright (c) 2026 Chris Barlow. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Barlow
+-/
+import Mathlib.Tactic.Ring
+
+theorem n4_plus_one_factor_over_sqrt_shift (n : ℤ) : (2 * n ^ 2 - 2 * n + 1) ∣ (4 * n ^ 4 + 1) :=
+  ⟨2 * n ^ 2 + 2 * n + 1, by ring⟩
